packages/beacon-node/test/sim/withdrawal-interop.test.ts

```diff
@@ -217,6 +219,9 @@ describe("executionEngine / ExecutionEngineHttp", function () {
     // delay a bit so regular sync sees it's up to date and sync is completed from the beginning
     const genesisSlotsDelay = 8;
 
+    // TODO for g11tech: Why 4? Provide rationale for the number
```

Comment:
@g11tech: Why 4? Provide rationale for the number

Reply:
from the test length run i noted that if there is no issue, withdrawal blocks come out to be 6, so i chose 4 as safe number incase there are any skipped slots for whatever reasons (sometimes have seen it happening)